### PR TITLE
Upgrade chefdk version to 4.4.47

### PR DESCRIPTION
**Pull request by JohnRoesler**
# Description

Upgrade chefdk version to 4.4.47 to get latest fixes (inspec, e.g.)

## Issues Resolved

none

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Review by PowerSchill — DISMISSED** [review]

**Review by PowerSchill — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -16,7 +16,17 @@ Then use the orb in a workflow.
...
 
-### Version 1.1.2
+### Version 2.2.0
+
+The sous-chef standard for running platform tests is `kitchen/dokken-single` for every platform
+and each test should require `lint_and_unit` to allow circle to perform parallel builds.
+
+To generate your circle yaml config utilize:
+
+1. Alias `dokken` to `KITCHEN_LOCAL_YAML=kitchen.dokken.yml kitchen`
```
Comment (damacus) [inline README.md:26]: Use `-` instead of a numbered list, this looks a bit weird

**Review by xorima — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -26,9 +36,9 @@ lint_and_unit: &lint_and_unit
   - lint-yaml
   - lint-markdown
 
-version: 2.1
+version: 2.1  # this is circle version
```
Comment (damacus) [inline README.md]: As discussed in Slack, these comments are a little bit 😞 
Should we link to the CircleCI documentation instead?

**Review by damacus — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JohnRoesler) [reply README.md:26]: this is a set of steps, so I think it makes more sense to have numbers rather than a non-ordered list

Comment (damacus) [reply README.md:26]: 🤔 ai true, except in markdown, they're all number 1) 1) 1) which doesn't make sense without a markdown previewer

**Review by PowerSchill — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JohnRoesler) [reply README.md:26]: markdown lint doesn't like it when you actually use the numbers vs 1111 😂

**Review by damacus — APPROVED** [review]

**Pull request merged**